src/coreclr/src/inc/clr_std/type_traits

```diff
@@ -186,6 +186,70 @@ namespace std
         typedef _Ty1 type;
     };
 
+    //-----------------------------------------------------------------------------------------
```

Comment:
I am not sure if these methods should be in src/inc or should stay in jitstd (probably in utility.h?).

Reply:
I moved `make_signed/unsigned` to clr_std/type_traits because it's where they live in STL.
If we compile with `USE_STL` defined we'll include standard <type_traits> which has `make_signed/unsigned`